internal/k8s/context.go

```diff
@@ -3,6 +3,8 @@ package k8s
 import (
 	"fmt"
 
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
@paivagustavo v12? How about metav1 to keep consistency?

Reply:
I guess Goland didn't help me here 😅. Changed :)